extensions/table/table_entry.go

```diff
@@ -12,13 +14,15 @@ TableEntry represents an entry in a table test.  You generally use the `Entry` c
 type TableEntry struct {
 	Description string
 	Parameters  []interface{}
-	Pending     bool
-	Focused     bool
+	Flag        types.FlagType
```

Comment:
I also first thought it's a good idea to use the single `Flag` field instead of `Pending` and `Focused`. I'm not sure since I haven't written the original code but it looks like the `Pending` and `Focused` fields are there on purpose since they are exported to the user?
Also, if we use do this change, we will break all users who construct a `TableEntry` directly (rather than using the `(F/P/X)Entry` constructors). Looking at the go docs for [TableEntry](https://github.com/onsi/ginkgo/blob/master/extensions/table/table_entry.go#L12), they state

> TableEntry represents an entry in a table test. You **generally** use the `Entry` constructor.

Although I've not seen people constructing the `TableEntry` directly, some people might do in which case we would break them with these changes.